{"uuid": "4cacbb92-67fe-4c7c-ae0d-b726b350274e", "children": ["d989c8ff-8d09-473f-a485-34de86d64567"], "befores": [{"name": "project_data", "status": "passed", "start": 1746032293406, "stop": 1746032293406}], "afters": [{"name": "project_data::0", "status": "broken", "statusDetails": {"message": "ValueError: Unexpected status code: 404\n", "trace": "  File \"C:\\Apps\\teamcity-tests\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"C:\\Apps\\teamcity-tests\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n    ~~~~^^^^\n  File \"C:\\Apps\\teamcity-tests\\conftest.py\", line 26, in project_data\n    super_admin.api_manager.project_api.clean_up_project(project_id)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^\n  File \"C:\\Apps\\teamcity-tests\\api\\project_api.py\", line 34, in clean_up_project\n    self.send_request('DELETE', f'/app/rest/projects/id:{created_project_id}', expected_status=204)\n    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Apps\\teamcity-tests\\custom_requester\\custom_requester.py\", line 23, in send_request\n    raise ValueError(f'Unexpected status code: {response.status_code}')\n"}, "steps": [{"name": "Clean up projects with id yjPSqSXKMsm", "status": "broken", "statusDetails": {"message": "ValueError: Unexpected status code: 404\n", "trace": "  File \"C:\\Apps\\teamcity-tests\\api\\project_api.py\", line 34, in clean_up_project\n    self.send_request('DELETE', f'/app/rest/projects/id:{created_project_id}', expected_status=204)\n    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Apps\\teamcity-tests\\custom_requester\\custom_requester.py\", line 23, in send_request\n    raise ValueError(f'Unexpected status code: {response.status_code}')\n"}, "steps": [{"name": "Sending DELETE request to http://localhost:8111/app/rest/projects/id:yjPSqSXKMsm with data: None, expected_status: 204", "status": "broken", "statusDetails": {"message": "ValueError: Unexpected status code: 404\n", "trace": "  File \"C:\\Apps\\teamcity-tests\\custom_requester\\custom_requester.py\", line 23, in send_request\n    raise ValueError(f'Unexpected status code: {response.status_code}')\n"}, "start": 1746032297831, "stop": 1746032297837}], "start": 1746032297831, "stop": 1746032297838}], "start": 1746032297831, "stop": 1746032297840}], "start": 1746032293406, "stop": 1746032297840}